doc/1-intro/faq.md

```diff
@@ -0,0 +1,138 @@
+
+### will Furnace support the Nintendo 64?
+
+the N64 lacks any form of audio synthesizer chip. many games use MIDI or XM or other such formats internally, but everything is mixed in software and sent to a simple stereo DAC.
+
+### will Furnace support this obscure PCM-only chip?
+
+probably not, as with very few exceptions these are effectively all the same.
+
+### will Furnace support the Roland MT-32?
+
+no. MT-32 is used with MIDI in 99.999% of situations. it lacks a direct register interface.
+
+also, Furnace is not a MIDI tracker....
+
+## importing
+
+### will Furnace import MIDI files?
+
+nope. Furnace is not a MIDI tracker.
+
+### why does this imported file sound wrong?
+
+Furnace makes every attempt to import non-Furnace files as accurately and completely as possible, but there are fundamental differences between formats that cannot be directly translated. an import from a non-Furnace format should always be considered the starting point of a conversion, not a final product.
+
+### can I import VGM or NSF?
+
+nope. it's a feature that's been requested many times, but there are no plans to implement it.
+
+for NSF import, you can use [a modified version of FamiTracker called NSFImport](http://rainwarrior.ca/projects/nes/nsfimport.html) and then import the resulting .ftm into Furnace.
+it's all speed 1 though, so don't expect any songs to be nicely laid out with instruments and all.
+
+### how can I use an SF2 soundfont?
+
+one way is to use [OpenMPT](https://openmpt.org/) to open the SF2 file, and save WAV files from there.
+
+### how do I import instruments from this SNES game?
+
+use [split700](https://github.com/gocha/split700) to extract the BRR samples from an SPC. there is presently no way to import envelopes or other parameters.
+
+### how do I import instruments from this Sega Genesis game?
+
+extract FM patches from a VGM file using [vgm2pre](https://github.com/vgmtool/vgm2pre) or similar tool. bear in mind that these are only the parameters for the FM synth, and the way the instrument is heard in-game may include pitch bends or other effects that can't be extracted.
+
+for PSG instruments, see the next question.
+
+### how do I import instruments from this NES/SMS/GB/C64/etc. game?
+
+PSG chips (such as those in the systems mentioned) don't have any inherent concept of instruments or patches. all of that is handled in software, and each sound driver has its own way of doing things. generally, the only option is to recreate the instrument from scratch.
+
+## exporting
+
+### will Furnace export MIDI files?
+
+nope. Furnace is not a MIDI tracker.
+
+### why does this exported VGM sound weird when I play it in other software?
+
+just as Furnace offers a choice of emulation cores, VGM players may use different cores with varying degrees of accuracy. also, some aspects of a song may not be supported by the VGM format, such as chip clock speeds or mixer levels.
```

Comment:
Mixer levels are supported by VGM format. It's just very inconsistent across players.